core/src/test/java/com/datastrato/gravitino/storage/kv/TestEntityKeyEncoding.java

```diff
@@ -55,6 +56,7 @@
 import org.mockito.Mockito;
 
 @TestInstance(Lifecycle.PER_CLASS)
+@Disabled
```

Comment:
You should add the `Disabled` reason here and there.

Reply:
done